test/integration-tests/IntegrationTests.Helpers/TestHelper.cs

```diff
@@ -79,29 +78,32 @@ public async Task<Container> StartContainerAsync(int traceAgentPort, int webPort
 
         Output.WriteLine($"Container was started successfully.");
 
-        PowershellHelper.RunCommand($"docker exec {container.Name} curl -v {healthCheckEndpoint}", Output);
+        PowershellHelper.RunCommand($"docker exec {container.Name} curl -v {agentHealthzUrl}", Output);
 
-        var healthChecksUrl = $"http://localhost:{webPort}/healthz";
+        var webappHealthzUrl = $"http://localhost:{webPort}/healthz";
 
         // try healthz a few times
         HttpStatusCode statusCode = default;
         HttpClient client = new();
 
         for (int retry = 0; retry < 5; retry++)
         {
-            var response = await client.GetAsync(healthChecksUrl);
+            var response = await client.GetAsync(webappHealthzUrl);
             statusCode = response.StatusCode;
 
             if (statusCode == HttpStatusCode.OK)
             {
                 break;
             }
 
+            Output.WriteLine($"IIS App healthz retry {retry + 1}/5");
```

Comment:
FYI for the last iteration, it will log but not retry 😅

Reply:
should replace `retry` with `failed` I guess.